mlir-sycl/lib/Dialect/IR/SYCLOpsDialect.cpp

```diff
@@ -124,34 +124,33 @@ class SYCLOpAsmInterface final : public mlir::OpAsmDialectInterface {
 mlir::OpAsmDialectInterface::AliasResult
 SYCLOpAsmInterface::getAlias(mlir::Type Type, llvm::raw_ostream &OS) const {
   return llvm::TypeSwitch<mlir::Type, AliasResult>(Type)
-      .Case<mlir::sycl::AccessorType>([&](auto Ty) {
-        OS << "sycl_" << decltype(Ty)::getMnemonic() << "_" << Ty.getDimension()
-           << "_" << Ty.getType() << "_" << getAlias(Ty.getAccessMode()) << "_"
-           << getAlias(Ty.getTargetMode());
-        return AliasResult::FinalAlias;
-      })
-      .Case<mlir::sycl::ItemType, mlir::sycl::ItemBaseType>([&](auto Ty) {
-        OS << "sycl_" << decltype(Ty)::getMnemonic() << "_" << Ty.getDimension()
-           << "_";
-        return AliasResult::OverridableAlias;
-      })
+      .Case<mlir::sycl::AssertHappenedType, mlir::sycl::BFloat16Type>(
+          [&](auto Ty) {
+            OS << "sycl_" << decltype(Ty)::getMnemonic() << "_";
```

Comment:
No need for trailing "_".

Reply:
There is a need for trailing "_", as there are mnemonic like `bfloat16`, which ends with number.

Reply:
True!